ios/brave-ios/Sources/Brave/Frontend/UserContent/UserScripts/Scripts_Dynamic/ScriptHandlers/Paged/PlaylistScriptHandler.swift

```diff
@@ -207,7 +207,7 @@ class PlaylistScriptHandler: NSObject, TabContentScript, TabObserver {
       // When the download starts, it uses the database ID.
       // If we suddenly change the ID, downloads and updates get out of wack
       var item = item
-      item.tagId = $0  // Use the ID that it was saved as in the database, rather than the Javascript ID
+      item.tagId = $0
```

Comment:
Please restore this comment. Just place it on a separate line like the linter recommends